### DISCUSSION TITLE
picking the value of a field from a filtered array

### DISCUSSION THREAD

**Discussion by vumajc** [antonmedv/expr#445] [Q&A]
How can I get just the value of the field out of a filtered array. In the playground example, I am trying to do this
```Posts | filter(.ID == 1) | first() | .Title```

and end up with 
```
unexpected token Operator(".") (1:38)
 | Posts | filter(.ID == 1) | first() | .Title
 | .....................................^
```

**Comment by antonmedv** [comment]
(Posts | filter(.ID == 1) | first()).Title


Or 


    let head = Posts | filter(.ID == 1) | first(); head.Title
